src/helm/benchmark/run.py

```diff
@@ -122,6 +133,18 @@ def add_run_args(parser: argparse.ArgumentParser):
         default=None,
         help="Skip creation of instances (basically do nothing but just parse everything).",
     )
+    parser.add_argument(
+        "--cache-instances",
+        action="store_true",
+        default=None,
+        help="Save generated instances input to model to disk. If already cached, read instances from file.",
+    )
+    parser.add_argument(
+        "--cache-instances-only",
```

Comment:
I think that in `validate_args`, we should check that if `cache-instances-only` is set, `cache-instances` should also be set.

Reply:
Done.